packages/replay-next/src/utils/evaluate.ts

```diff
@@ -0,0 +1,43 @@
+import { FrameId, PauseId } from "@replayio/protocol";
+
+import { ThreadFront } from "protocol/thread/thread";
```

Comment:
:hand: Can we avoid continuing to reference `ThreadFront` here? I'd be fine with forcing us to pass in `pauseId` all the time.